sql/src/main/java/io/crate/exceptions/InvalidColumnNameException.java

```diff
@@ -21,27 +21,10 @@
 
 package io.crate.exceptions;
 
-import io.crate.metadata.TableIdent;
-
-import java.util.Collections;
 import java.util.Locale;
 
-public class InvalidColumnNameException extends ValidationException implements TableScopeException {
-
-    private final TableIdent tableIdent;
-
-    public InvalidColumnNameException(String columnName, TableIdent tableIdent) {
-        super(String.format(Locale.ENGLISH, "column name \"%s\" is invalid.", columnName));
-        this.tableIdent = tableIdent;
-    }
-
-    @Override
-    public int errorCode() {
-        return 2;
-    }
-
-    @Override
-    public Iterable<TableIdent> getTableIdents() {
-        return Collections.singletonList(tableIdent);
+public class InvalidColumnNameException extends RuntimeException {
```

Comment:
you still need to derive from CrateException in order to keep the errorcode and also for the further wrapping etc. best would probably be ClusterScopeException